packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
       negatable: true,
       defaultsTo: false,
-      help: 'Generate a Flutter plugin project.'
+      help: 'Generate a Flutter package project.'
```

Comment:
We should have a bit more verbose description here, to help the user choose between a regular project, and plugin project, and a package project.